### ISSUE TITLE
Have the professor check and see if this works

### ISSUE THREAD

**Issue by CakeJamble** [UC-Irvine-CS175/final-project-saddleback#8]
Question: Can a non-member, admin of the channel access and edit the board?

If you can see this issue, try and access the project board.
If you can see that too, let me know what it looks like. You can reply with a screenshot or just describe it as a comment.

![Image](https://user-images.githubusercontent.com/104522681/248584182-2ec9840d-ce70-4fa8-9e39-997fb418e686.png)

Click through to see the project. You should also be able to add new items to the board too. If it works, move the item related to this issue to Done. 😃

Thanks,
Jake

**Comment by nadia-eecs** [comment]
@campjake i can see it but can a non-admin make one and respond to it?

**Comment by CakeJamble** [comment]
@nadia-eecs 
Yep! This project board was created by a non-admin member account I added to the group (see pic below). It was copied from a Template Project that organization members can read, but only organization owners can write to/are admins of.

![test_project](https://github.com/UC-Irvine-CS175/final-project-saddleback/assets/104522681/83558a89-3a40-4ad7-a5ee-cb2ceea3f304)


I also made a [separate issue](https://github.com/UC-Irvine-CS175/final-project-saddleback/issues/7) for other members of the @UC-Irvine-CS175/saddleback  Team to try and update for themselves.

Despite not being added as a member of the team, you (and other owners) seem to have read/write access over all Project Boards in an organization (yay! 😄 )
